src/Pelagos/Bundle/AppBundle/Controller/DefaultController.php

```diff
@@ -22,7 +22,11 @@ class DefaultController extends Controller
      */
     public function indexAction()
     {
-        return $this->render('PelagosAppBundle:Default:index.html.twig');
+        if ($this->get('kernel')->isDebug()) {
+            return $this->render('PelagosAppBundle:Default:index.html.twig');
+        } else {
+            return $this->redirect('/', 302);
```

Comment:
Should this be 301?

Reply:
301 is permanent redirect, i'm not sure how that might behave.